scripts/fetch.sh

```diff
@@ -113,9 +114,17 @@ sync_references() {
   if [ -z "${rev_list}" ]; then
     echo "skipping module ${owner}/${repo}, no references to sync"
   else
+    skippable_refs="{}"
+    if [ -f "${mod_skip_refs_file}" ]; then
+      skippable_refs="$(cat "${mod_skip_refs_file}")"
+    fi
     for reference in $(echo "${rev_list}"); do
-      echo "processing reference ${owner}/${repo}:${reference}"
-      process_ref "${reference}"
+      if [ "$(echo "${skippable_refs}" | jq --arg jqref "${reference}" 'has($jqref)')" == "true" ]; then
```

Comment:
I'm sure there's a more elegant way to do this but I couldn't get it to work with 
```suggestion
      if $(echo "${skippable_refs}" | jq --arg jqref "${reference}" 'has($jqref)'); then
```
which returns false but still skips it. Some help from folks more talented with bash than me, is welcome.

Reply:
Maybe just me but when scripts get to this level of complexity I prefer writing them as a go command (which could have tests and first class data structures).

I think it might have worked as you listed if you pass this flag, but it may not work with `set -e`:
```
-e, --exit-status         set exit status code based on the output;
```

Reply:
Agree 💯 I'd rather have this as go code to maintain instead of bash. I'll merge as it is now and hopefully let's tackle migrating this script to go as a follow up.